packages/RawButton/test/RawButton.spec.js

```diff
@@ -0,0 +1,128 @@
+/* eslint-disable jsx-a11y/click-events-have-key-events, jsx-a11y/no-static-element-interactions */
+
+import React from "react";
+import { render, fireEvent } from "react-testing-library";
+import RawButton from "../src";
+
+const noop = () => {};
+
+const pressEnter = { key: "Enter" };
+const pressSpace = { key: " " };
+
+function renderComponent(props = {}) {
+  const defaultProps = {
+    onClick: noop,
+  };
+
+  return render(
+    <RawButton {...defaultProps} {...props}>
+      {props.children || "happy button"}
+    </RawButton>
+  );
+}
+
+describe("RawButton", () => {
+  it("should render with default props", () => {
+    const { getByText, getByRole, container } = renderComponent();
+
+    expect(getByText(/happy button/i)).toBeVisible();
```

Comment:
So with styled-components `toBeVisible` is actually able to handle visibility because it's CSS-in-JS right?

Reply:
These tests pass, so... yes?
I'm not really sure that `toBeVisible()` is the right assertion here.  Really, I want to know that it exists in the DOM... is there a different function I should use instead?

Reply:
`toBeInTheDocument`.
That was my question here: `toBeVisible` looks at `visibility: hidden` etc. to check actual visibility. This usually doesn't work with SCSS since we mock those styles out in tests. But with CSS-in-JS it sounds like it should work out

Reply:
https://github.com/testing-library/jest-dom you can see all the assertions.

Reply:
I also suggest to use `toBeInTheDocument`